File: pret/Views/ComptePretView.cs
using System;

namespace Pret.Views
{
    public class ComptePretView
    {
        public int IdCompte { get; set; }
        public int IdClient { get; set; }

        // public string ClientNom { get; set; }
        public string Libelle { get; set; }

        // public string ClientPrenom { get; set; }

        public DateOnly DateOuverture { get; set; }
        public decimal CapitalEmprunte { get; set; }
        public decimal TauxInteret { get; set; }
        public int DureeMois { get; set; }
        public DateOnly? DateEcheance { get; set; }
        public string Statut { get; set; }
        public decimal Solde { get; set; } = 0.0m;
    }
}

File: server-ejb/src/main/java/com/example/views/ComptePretView.java
package com.example.views;

import lombok.Data;
import java.math.BigDecimal;
import java.time.LocalDate;

@Data
public class ComptePretView {
    private int idCompte;
    private int idClient;
    private String libelle;
    private LocalDate dateOuverture;
    private BigDecimal capitalEmprunte;
    private BigDecimal tauxInteret;
    private int dureeMois;
    private LocalDate dateEcheance;
    private String statut;
    private BigDecimal solde = BigDecimal.ZERO;
}
